_config.yml

```
@@ -959,3 +959,10 @@ podcast:
           url: "https://artsy.github.io/blog/2016/01/30/iOS-Junior-Interviews/"
         - title: "Artsy jobs"
           url: "https://www.artsy.net/jobs"
+    - title: "3: WAYAHDYGH: Chris"
+      date: "2021-01-19 16:31:14 -0500"
+      description: "Matt Dole asks engineer Chris Pappas who he is and how he got here. Tune in to learn about BBSes and how skateboarding started Chris' career in software."
```

```suggestion
      description: "Matt Dole asks engineer Chris Pappas who he is and how he got here. Tune in to learn about BBSes and how skateboarding started Chris's career in software."
```

`Chris'` = belonging to multiple Chrises. `Chris's` = belonging to this Chris.